qemu/tests/block_copy.py

```diff
@@ -425,3 +430,27 @@ def verify_md5(self, file_name):
         if status != 0:
             raise error.TestFail("File %s changed, md5sum check output: %s" %
                                  (file_name, output))
+
+    def hot_unplug(self):
```

Comment:
@xutian thanks.
1. self.device in this class is only a string, I need to get the device instance to call function unplug(), and here I use qcontainer.DevContainer.simple_unplug() than qdevices.QBaseDevice.unplug() because it is safer. And the function also check the block job status after unplug the device.
2. I have several test cases from different block job features have same step like unplug the device during the block job. So add it here fore sharing the code for block commit, stream and mirror.